lib/api/user_repo.dart

```diff
@@ -5,34 +5,44 @@ import 'package:stronk/domain/model/user.dart';
 
 abstract class UserRepository {
   Future<User> retrieveUser();
+
+  Future<User> retrieveUserById(String id);
 }
 
 class UserRepositoryImpl implements UserRepository {
   GraphQLUtility utility;
 
-  UserRepositoryImpl({@required GraphQLUtility utility});
+  UserRepositoryImpl({@required this.utility});
 
   @override
   Future<User> retrieveUser() async {
```

Comment:
This is being used by the request_middleware.dart file and I wasn't sure whether to remove it or not.

Reply:
uh could you clarify a bit :)